lte/gateway/c/session_manager/ShardTracker.cpp

```diff
@@ -13,45 +13,43 @@
 
 #include <string>
 #include <vector>
+#include <set>
 
 #include "ShardTracker.h"
 
 namespace magma {
 
-int ShardTracker::add_ue() {
-  /*
-  * If there are no shards, add an entry for the shard
-  * representing a single UE at the new shard
-  */
-  if (ue_count_per_shard_.size() == 0) {
-    ue_count_per_shard_.push_back(1);
-    return 0;
+uint16_t ShardTracker::add_ue(const std::string imsi) {
```

Comment:
Sorry to say it again burt I don't like this design: now we have 2 places where to maintain imsis.

What we really need either now or when we do rebalance is to know what shard a UE is at and how many UEs are per shard. 

Per the design document we are not ready yet for rebalancing, so let's not do optimizations on something we don't even want to discuss.